eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/Types.hs

```diff
@@ -187,15 +184,13 @@ instance
   DecCBOR (EpochState era)
   where
   decCBOR =
-    decodeRecordNamed "EpochState" (const 6) $
+    decodeRecordNamed "EpochState" (const 4) $
       flip evalStateT mempty $ do
         esAccountState <- lift decCBOR
-        esLState <- decSharePlusCBOR
         esSnapshots <- decSharePlusCBOR
-        esPrevPp <- lift decCBOR
-        esPp <- lift decCBOR
+        esLState <- decSharePlusCBOR
```

Comment:
This should go above the `esSnapshots`. It is important for sharing when decoding
```suggestion
```

Maybe we should change the order in the `EpochState` now as well, since this PR is breaking the type anyways?